src/pages/Home.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable max-len */
```

Comment:
- [ ]  Let's name disable linter rules. these are here to help us write a clean block of code.